extensions/mssql/src/objectManagement/localizedConstants.ts

```diff
@@ -57,6 +57,7 @@ export const labelSelectFolder = localize('objectManagement.labelSelectFolder',
 export const DataFileLabel = localize('objectManagement.dataFileLabel', "Data");
 export const LogFileLabel = localize('objectManagement.logFileLabel', "Log");
 export const BackButtonLabel = localize('objectManagement.backButtonLabel', "Back");
+export const SchemaIsEmptyInfoMessage: string = localize('objectManagement.schemaIsEmptyInfoMessage', "The selected schema does not contain any objects.");
```

Comment:
We can select other objects besides ones under a specific schema. Could you make this message more generic? Maybe something like "No securable objects could be found for the specified selection"?